src/data/chargen/books/ninth-world-guidebook

```diff
@@ -0,0 +1,227 @@
+var descriptors = [
+  {
+    name: 'Coraoan',
+    sourcebook: 'Ninth World Guidebook',
+    page: 215,
+    sources: [
+      { sourcebook: 'Ninth World Guidebook', page: 215 }
+    ],
+    stats: {
+      speed: 3
+    },
+    shins: 10,
+    skills: {
+      trained: [
+        'All tasks related to the history of Corao and the surrounding regions',
+        'All tasks having to do with the numenera'
+      ]
+    }
+  },
+  {
+    name: 'Desert-Dwelling',
+    sourcebook: 'Ninth World Guidebook',
+    page: 215,
+    sources: [
+      { sourcebook: 'Ninth World Guidebook', page: 215 }
+    ],
+    stats: {
+      might: 4
+    },
+    shins: 10,
+    skills: {
+      trained: [
+        'Any task involving resisting damage from heat',
+        'All tasks having to do with finding food or water'
+      ],
+      inability: [
+        'Any task involving resisting damage from cold, and you have +2 to Armor that applies only to damage from heat',
+        'Any task involving swimming or handling watercraft'
+      ]
+    },
+    extras: [
+      'Long-suffering: You can go twice as long without food and water as another human.'
+    ]
+  },
+  {
+    name: 'Devout',
+    sourcebook: 'Ninth World Guidebook',
+    page: 216,
+    sources: [
+      { sourcebook: 'Ninth World Guidebook', page: 216 }
+    ],
+    stats: {
+      intellect: 3
+    },
+    shins: 10,
+    extras: [
+      'Steadfast: Your faith gives you the foundation to deal with stress. When something would alter the difficulty of your action to your detriment, you can negate one step of the modification. Once you use this ability, you cannot use it again until after your next ten-hour recovery roll.'
+    ]
+  },
+  {
+    name: 'Echryni',
+    sourcebook: 'Ninth World Guidebook',
+    page: 216,
+    sources: [
+      { sourcebook: 'Ninth World Guidebook', page: 216 }
+    ],
+    shins: 10,
+    skills: {
+      trained: [
+        'All tasks involving positive social interactions'
+      ],
+      inability: [
+        ' If you go more than 28 hours without switching between water breathing and air breathing, you gain an inability in movement-related tasks, including running, jumping, and climbing. The inability persists until you make the switch between environments for at least a few minutes.'
+      ]
+    },
+    extras: [
+      'Gregarious: Despite your unusual appearance, your social skills charm even the most standoffish of creatures. You are trained in all tasks involving positive social interactions.',
```

Comment:
I'm concerned these exceed the limits of what MCG allow for generators, as it contains basically the entire descriptive text. Stripping them back to a one sentence summary would probably be advised.